ga4gh/frontend.py

```diff
@@ -134,7 +134,10 @@ def getReferenceSet(id):
 
 @app.route('/callsets/search', methods=['POST'])
 def searchCallSets():
-    raise frontendExceptions.PathNotFoundException()
+    if flask.request.method == "POST":
```

Comment:
It might be good to do some refactoring here if we're going to have this same if POST then handleHttpPost else handleHttpOptions pattern in all of our endpoints.  I'm not sure if we want to do that in this PR or another one.